### DISCUSSION TITLE
When did the config mechanism changed?

### DISCUSSION THREAD

**Discussion by eko234** [kakoune-lsp/kakoune-lsp#773] [Q&A]
Hemlo, I'm updating kak-lsp, and afict, it seems now we config stuff with hooks, it took me a time to notice :V

**Comment by krobelus** [comment]
How did you even notice? Everything should keep working as before, I'm not aware of breaking changes except the edge cases mentioned in the changelog. LMK if it broke somehow.

The config change was merged a couple days ago.
We're still looking for a nice way to sent individual keys like `root` without setting all the others. So if there is a good reason, there might be a breaking change (but not yet for kak-lsp.toml users, that should work as before)
